### DISCUSSION TITLE
cn/2024/12/19/missed-love-and-romance/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#200] [Announcements]
# cn/2024/12/19/missed-love-and-romance/

https://hongtaoh.com/cn/2024/12/19/missed-love-and-romance/

**Comment by yuzhao66** [comment]
太实在了，师兄你真没把我们当外人哈哈

**Reply by hongtaoh** [reply]
这篇博客我反反复复改了很多遍，要在隐私和信息之间保持一个平衡。看你的回复，貌似我还是暴露了太多隐私。。

**Reply by yuzhao66** [reply]
哈哈，曲折中上升，我觉得后面说不定有看了你的博客联系你的哈
